### ISSUE TITLE
Goose is non-functional in Python 3

### ISSUE THREAD

**Issue by fake-name** [grangier/python-goose#148]
Title is largely self explanatory. 

Primary limitation seems to be reliance on BeautifulSoup 3, which has been EOL for quite a while now, and really should be migrated away from.

**Comment by vetal4444** [comment]
Prepare PR to add py3 support: https://github.com/grangier/python-goose/pull/220

**Comment by xanderdunn** [comment]
+1 for this.  Why uses Python 2!?

**Comment by hipoglucido** [comment]
Still waitting for Python 3 support :)

**Comment by hnykda** [comment]
I believe this project is dead. Use https://github.com/codelucas/newspaper instead, which is inspired by goose and supports Python 3 flawlessly.

**Comment by hipoglucido** [comment]
Yep, I already knew it but I just wanted to do some comparison of the available tools. Indeed, I will use it. Thanks!

**Comment by float64co** [comment]
Any plans to introduce Python 3 support to this project?

**Comment by float64co** [comment]
Any plans to introduce Python 3 support to this project?

**Comment by lababidi** [comment]
Hi everyone, this may come off as self promotion, but I went ahead and forked goose to work with python3. http://github.com/goose3/goose3 Enjoy
